example/radar-chart.md

```diff
@@ -0,0 +1,34 @@
+# Radar Chart
+
+[Radar](http://d3plus.org/docs/#Radar) chart are a way of comparing multiple quantitative variables. This makes them useful for seeing which variables have similar values or if there are any outliers amongst each variable. 
```

Comment:
Radar chart**s**

Reply:
This comment was actually referring to the sentence description, it should read "Radar charts are", and the title should be "Radar Chart"